### ISSUE TITLE
`wp_calculate_image_srcset` not always array

### ISSUE THREAD

**Issue by curtismchale** [udx/wp-stateless#644]
The result of this function is not always an array. With debugging on I get a bunch of errors and can't save posts/pages/cpts properly.

I'm not sure if this is a complete fix but the following code does resolve the debugging issues:

```php
        if ( is_array( $sources ) ){
          return array_filter($sources);
        } else {
          return null;
        }
```

It's inserted at line 501 in `wp-stateless/lib/classes/class-bootstrap.php`

**Comment by planv** [comment]
Fixed, will be released in version `3.2.3`

**Comment by planv** [comment]
Released. Closing the ticket.
Please let us know if you have any further questions
